package.json

```diff
@@ -2,26 +2,29 @@
   "name": "attr-accept",
   "description": "JavaScript implementation of the \"accept\" attribute for HTML5 <input type=\"file\">",
   "version": "0.0.0-development",
-  "main": "dist/index.js",
-  "module": "dist/es/index.js",
+  "source": "src/index.mjs",
```

Comment:
TIL parcel gets configured via a `"source"` field.

I also expected to see `"type": "module",` — is this worth adding? Then you wouldn't need to rename the `.js` files to `.mjs`. I believe `"main"` and `"module"` would then change to `"browser"` and `"main"` respectively.

Reply:
`browser` is specific to webpack iirc. Doing our `exports` as I did in the package.json should alleviate the need for that since it would be predicated on how it's being referenced i.e. `require` vs. `import`